workflow/Snakefile

```diff
@@ -49,6 +59,4 @@ rule all:
 
 
 localrules:
-    irf,
-    plot_irf,
     link_paths,
```

Comment:
Since you are at it, remove `link_paths`, that rule is nonexistent.

Reply:
Ok